workloads.json

```diff
@@ -0,0 +1,10 @@
+{
+  "microsoft.net.sdk.android": "32.0.447/6.0.300",
+  "microsoft.net.sdk.ios": "15.4.414/6.0.400",
+  "microsoft.net.sdk.maccatalyst": "15.4.414/6.0.400",
+  "microsoft.net.sdk.macos": "12.3.414/6.0.400",
+  "microsoft.net.sdk.maui": "6.0.400/6.0.300",
+  "microsoft.net.sdk.tvos": "15.4.414/6.0.400",
+  "microsoft.net.workload.mono.toolchain": "6.0.6/6.0.300",
+  "microsoft.net.workload.emscripten": "6.0.4/6.0.300"
+}
```

Comment:
The path to this file should be in `$(XamarinDotNetWorkloadSource)` above.

But I'd also make this change in a separate PR, and use this `.json`:
```json
{
  "microsoft.net.sdk.android": "32.0.447/6.0.400",
  "microsoft.net.sdk.ios": "15.4.442/6.0.400",
  "microsoft.net.sdk.maccatalyst": "15.4.442/6.0.400",
  "microsoft.net.sdk.macos": "12.3.442/6.0.400",
  "microsoft.net.sdk.maui": "6.0.424/6.0.400",
  "microsoft.net.sdk.tvos": "15.4.442/6.0.400",
  "microsoft.net.workload.mono.toolchain": "6.0.7/6.0.300",
  "microsoft.net.workload.emscripten": "6.0.4/6.0.300"
}
```
So we get the latest packs.

@jpobst also mentioned in the past, we try to use a matching build for Xamarin.Android. Do we need to get a `.pkg` from the .NET 6 release branch?